src/components/LoansByCategory/CategoryRow.vue

```diff
@@ -3,14 +3,24 @@
 		<div class="row title-row">
 			<div class="column small-12">
 				<h2 class="category-name">
-					<a class="view-all-link"
+					<router-link
+						class="view-all-link"
+						:to="cleanUrl"
+						:title="`View all ${cleanName} loans`"
+						v-kv-track-event="[
+							'Lending',
+							'click-Category-View-All',
+							`View all ${cleanName} loans`]">
+						{{ cleanName }} <span v-if="showViewAllLink" class="view-all-arrow">&rsaquo;</span>
+					</router-link>
+					<!-- <a class="view-all-link"
 						:href="cleanUrl"
 						:title="`View all ${cleanName} loans`"
 						v-kv-track-event="[
 							'Lending',
```

Comment:
Should we just delete this commented out code since it's been changed from an <a> to a <router-link>?

Reply:
yes. sorry I missed that